index.html

```diff
@@ -305,20 +305,22 @@ <h2 class="about-title">About me</h2>
       get in touch.
     </p>
 
-    <form class="desktop-form" action="https://formspree.io/f/mzbooyno" method="post">
+    <form id="form1" class="desktop-form" action="https://formspree.io/f/mzbooyno" method="post">
       <div class="desktop-input-wrap">
         <input class="desktop-form-input" maxlength="30" type="text" name="First Name" required placeholder="First Name">
         <input class="desktop-form-input-2" maxlength="30" type="text" name="Last Name" required placeholder="Last Name">
       </div>
-      <input class="desktop-form-input-email" type="email" name="Email" required placeholder="Email Address">
+      <input id="user-email1" class="desktop-form-input-email" type="email" name="Email" required placeholder="Email Address">
+      <span class="displayNone redColor">Please enter a email with lowercase letters here</span>
```

Comment:
- Hey, you've done well, however, according to the project requirement, 
`the error message should show up close to the "submit" button`. This could be just above the submit button right under the text area. please, kindly move the span right under the text area to meet this part of the project requirement to have your project approved.

![info](https://user-images.githubusercontent.com/59397159/189317987-627256bb-ea2f-4356-ad74-b639d180229b.png)